lib/components/engine.rb

```diff
@@ -0,0 +1,6 @@
+require 'rails'
```

Comment:
This is to get the generator to be available from the gem inside the application. Had to look it up. Do you know of another way?